### PR TITLE
Have configure output its configuration

### PR DESCRIPTION
**Pull request by perarnau**
Add a small output at the end of configure print the entire
configuration in terms of features/flags, to help users understand what
worked.

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -227,3 +227,44 @@ AC_CONFIG_FILES([Makefile
...
+
+OpenCL:
+=======
+
+Active:  $HAVE_OPENCL
+CFLAGS:  $OPENCL_CFLAGS
+LDFLAGS: $OPENCL_LIBS
+
+ZE:
+
```
Comment (NicolasDenoyelle) [inline configure.ac:264]: Update: 
```
ZE:
===
```

**Review by NicolasDenoyelle — CHANGES_REQUESTED** [review]
See my comment.
Thanks for the feature!

**Comment by perarnau** [comment]
@NicolasDenoyelle should be good once all the builds are done.

**Comment by perarnau** [comment]
Fix #181

**Review by NicolasDenoyelle — APPROVED** [review]

**Pull request merged**